packages/react/src/utils/polymorphicComponent.test.tsx

```diff
@@ -143,3 +143,7 @@ const htmlAnchorRef = React.createRef<HTMLAnchorElement>();
     bananas
   </ConstrainedComponent>
 );
+
+test('polymorphicComponent', () => {
```

Comment:
I made a comment on https://github.com/dequelabs/cauldron/pull/1519/files#r1635327695, but it can be done in either PR (whichever lands first).